### PR TITLE
Time-Release Pills now NOREACT

### PR DESCRIPTION
**Pull request by Kurfursten**
This is something discussed in passing during the freeze, and I liked the idea a lot. Time release pills were already really niche, this gives them more utility. I foresee great potential and also lazy bombs.

### CODE REVIEW

File: html/changelogs/Kurfurst-pillbalance.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+author: Kurfurst
+delete-after: True
+changes:
+ - tweak: Time-release pills are now NOREACT.
+ - bugfix: Xenoarch machines can now have their panel opened and be moved about with a wrench. By extension, you can now deconstruct them and upgrade them.
```
Comment (Iamgoofball) [inline html/changelogs/Kurfurst-pillbalance.yml:5]: ???

Comment (Kurfursten) [inline html/changelogs/Kurfurst-pillbalance.yml:5]: I did a hotfix with no changelog earlier tonight

Comment (ihadtoregisterforthis) [inline html/changelogs/Kurfurst-pillbalance.yml:5]: I agree here, adding the simplest explanation on what does NOREACT mean will save time for everyone.

Comment (Kurfursten) [inline html/changelogs/Kurfurst-pillbalance.yml:5]: He commented line 5 though, he's obviously commenting on the xenoarch hotfix, not on line 4

There is a better explanation of what noreact does on the pill itself when examined anyway

**Comment by FudgePucker** [comment]
Wait so I can make polyacid pills that fuck someone up then fucks up who ever helps them?

**Comment by Intigracy** [comment]
:+1:

**Pull request merged**